src/org/usfirst/frc/team2706/robot/commands/GetTargets.java

```diff
@@ -0,0 +1,39 @@
+package org.usfirst.frc.team2706.robot.commands;
+
+import org.usfirst.frc.team2706.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ * A test command that uses the camera to get target information.
+ */
+public class GetTargets extends Command {
```

Comment:
If you are never planning on using initialize() interrupted() and end(), then just remove them from the class you don't need them, along with the constructor, maybe even put the Robot.camera.GetTargets(); in initialize if you want a response quicker from them command.